foundry.toml

```
@@ -0,0 +1,4 @@
+[profile.checker]
+test = 'test-forge'
```

Could also have been this and not have had the file 
```suggestion
test = 'lib/certora/checker'
```

---

It would be great, but I could not make it work that way. I think the issue was that `forge` would not recognize what to compile and what to test